use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,227 @@
+
+
+
+### Why we create init process?
+
+Currently, if you want to set up additional configs for Fluent Bit running on ECS, you need to [build a custom image](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file). And you cannot set up a [PARSER] section for Fluent Bit because using a parser in Fluent Bit requires you to change the Fluent Bit command or specify the path of the parser file under the [SERVICE] section.
+
+Init process is designed to solve above mentioned problems and improve user experience of using Fluent Bit on ECS. You only need to set the `ARN` or `Path` of the config file in the ECS Task Definition, without telling us if it is a parser file. Init process will process each config file, use [@INCLUDE](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/configuration-file#config_include_file-1) keyword to add them to the main config file, and change the Fluent Bit command if the parser is used.
+
+The init process also injects ECS Task Metadata into the Fluent Bit container as environment variables, so you can use them in the Fluent Bit config.
+
+
+
+### How to use multi-config feature?
+
+1. Use `aws-for-fluent-bit` images with `init` tag.
+
+2. Specify config files as environment variable in the FireLens configuration Environment setting.
+
+   **Example:**
+
+   ```
+   "environment": [
+                   {
+                       "name": "aws_fluent_bit_init_s3_1",
+                       "value": "arn:aws:s3:::yourBucket/aaaaa.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_s3_2",
+                       "value": "arn:aws:s3:::yourBucket/bbbbb.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_file_1",
+                       "value": "/ecs/s3.conf"
+                   }
+               ]
+   ```
+
+   **We support two kinds of config files:**
+
+   * Config files which stored in your S3 bucket
+     You need to set the name of the env var using prefix `aws_fluent_bit_init_s3_`, the number after this prefix cannot be repeated, and set the `ARN` of your config file as the value of the env var.
+
+   * Config files which from the image
+
+     * Use [our built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs)
+     
+       You need to set the name of the env var using prefix `aws_fluent_bit_init_file_`, the number after this prefix cannot be repeated, and set the `Path` of your config file as the value of the env var.
+     
+     * Build a custom image
+     
+       You can build a custom image using image with `init` tag as the base and add your own configs. For more details please see [this example](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/config-file-type-file).
```

Comment:
That example is not ideal to link to since it is for the old single config file experience, It'd be better to just give them the Dockerfile to do this since it is so short:

```
FROM {init tag} # i am being lazy
ADD extra.conf /extra.conf
ADD another.conf /another.conf
ADD parsers.conf /parsers.conf
```

Reply:
And then remind them they can just reference the `/parsers.conf` and other paths via the init env vars